tests/benchmarks/test_competitor_adapters.py

```
@@ -0,0 +1,1128 @@
+
+    assert not out_dir.exists()
+
+
+def test_cli_exits_2_for_manifest_with_excluded_competitor_entry(tmp_path: Path) -> None:
+    out_dir = tmp_path / "results" / "excluded-cli"
+    manifest_path = _write_yaml(
+        tmp_path / "competitors.yml",
+        {"competitors": [_valid_competitor_entry("gitmcp", eligibility={"status": "excluded"})]},
+    )
+
+    result = subprocess.run(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)